README.md

```diff
@@ -11,8 +11,8 @@ Console application developed to illustrate the programmatic control of YKUSH fa
 It executes one command per run, being appropriate to be executed as a console command.
 But it can be easily adapted to execute a work-flow with multiple commands and we encourage you to alter it to best fit your needs.
 
-The implementation makes use of libusb for Linux builds and hidapi for Windows.
-For Linux we include a build and installation script, `build.sh` and `install.sh` respectively, for building and installing the application. 
+The implementation makes use of libusb for Linux builds and hidapi for macOS and Windows.
+For Linux nad macOS we include a build and installation script, `build.sh` and `install.sh` respectively, for building and installing the application. 
```

Comment:
Small typo here, "nad" instead of "and".

Thanks for cleaning up my proof of concept mess! :)

Reply:
No problem - thanks for the proof of concept to get started.